python/paddle/v2/framework/__init__.py

```diff
@@ -1 +1,12 @@
+import os
+import core
 __all__ = ['proto']
+argv = ['paddle']
+if os.getenv('FLAGS_fraction_of_gpu_memory_to_use'):
```

Comment:
Parsing from environment variables is not needed since gflags has already done that.

Reply:
Done